target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -40,6 +40,11 @@ ipq40xx_setup_interfaces()
 			"0u@eth0" "1:lan" "2:lan" "3:lan" "4:lan"
 		;;
 	avm,fritzrepeater-1200|\
+	avm,fritzrepeater-3000|\
```

Comment:
I think this is a merge gone wrong? There are now two entries in 02_network for avm,fritzrepeater-3000 and the compex,wpj428.